net/nginx/Makefile

```diff
@@ -207,14 +209,40 @@ define Download/nginx-mod-geoip2
   PROTO:=git
 endef
 
+define Package/nginx-mod-lua-resty-core
+  $(call Package/nginx/default)
+  DEPENDS:=+nginx-mod-lua
```

Comment:
add lua-resty-lrucache as a dependency

Reply:
Adding also `+luajit2` gets rid of:

```
tmp/.config-package.in:24994:error: recursive dependency detected!
tmp/.config-package.in:24994:   symbol PACKAGE_luajit2 is selected by PACKAGE_nginx-mod-lua-resty-lrucache
tmp/.config-package.in:114130:  symbol PACKAGE_nginx-mod-lua-resty-lrucache is selected by PACKAGE_nginx-mod-lua-resty-core
tmp/.config-package.in:114116:  symbol PACKAGE_nginx-mod-lua-resty-core depends on PACKAGE_luajit2
For a resolution refer to Documentation/kbuild/kconfig-language.rst
subsection "Kconfig recursive dependency limitations"
```

Reply:
actually in theory -mod-lua should change the dependency to luajit2